src/ranking.jl

```diff
@@ -45,12 +42,11 @@ end
 
 
 """
-    ordinalrank(x; sortkwargs...)
+    ordinalrank(x; lt=isless, by=identity, rev::Bool=false, ...)
 
 Return the [ordinal ranking](https://en.wikipedia.org/wiki/Ranking#Ordinal_ranking_.28.221234.22_ranking.29)
-("1234" ranking) of an array. The `lt` keyword allows providing a custom "less
-than" function; use `rev=true` to reverse the sorting order.
-All items in `x` are given distinct, successive ranks based on their
+("1234" ranking) of an array. Supports the same keyword arguments as `sort(x; sortkwargs...)`
```

Comment:
`sortkwargs` isn't defined now. Same below.
```suggestion
("1234" ranking) of an array. Supports the same keyword arguments as the `sort`
```

Reply:
Actually, in L48 I sort-of introduce the *sortkwargs*, because otherwise later in the docstring it would not be entirely clear that the keyword args are passed through to `sort` to define the element order. I think from the context (*supports the same keyword args as ...*) it should be clear that *sortkwargs* refers to the keyword arguments of *ordinalranks()*. But if you have a better suggestion, I'm eager to fix it.

Reply:
OK. I would just have said "in the sorted vector" in the next sentence.

As you prefer, but if you keep this then please replace `sort(x)` with just `sort` below.

Reply:
Yes, that sounds good. I also simplified a bit the language in the other ranking docstrings.